subprojects/dependency-management/src/integTest/groovy/org/gradle/integtests/resolve/attributes/DependenciesAttributesIntegrationTest.groovy

```diff
@@ -20,9 +20,12 @@ import org.gradle.integtests.fixtures.GradleMetadataResolveRunner
 import org.gradle.integtests.fixtures.RequiredFeature
 import org.gradle.integtests.fixtures.ToBeFixedForInstantExecution
 import org.gradle.integtests.resolve.AbstractModuleDependencyResolveTest
+import org.gradle.integtests.fixtures.executer.GradleContextualExecuter
+import spock.lang.IgnoreIf
 import spock.lang.Issue
 import spock.lang.Unroll
 
+@IgnoreIf({ GradleContextualExecuter.vfsRetention })
```

Comment:
All the failing tests are only a problem for Windows, right? Can you use
```
@ToBeFixedForVfsRetention(because = "https://github.com/gradle/gradle/issues/13135", failsOnlyIf = TestPrecondition.WINDOWS)
```
instead?

I don't know if the tests fail because of https://github.com/gradle/gradle/issues/13135 or because of https://github.com/gradle/gradle/issues/13366. I suppose the logging test fail due to https://github.com/gradle/gradle/issues/13366. Any of those two issues seem to be a good `because` reason.